### PR TITLE
[Javascript][Closure] better code injection handling for Javascript (closure)API client

### PR DESCRIPTION
**Pull request by wing328**
For #3201

### CODE REVIEW

**Comment by wing328** [comment]
cc @achew22

**Comment by earth2marsh** [comment]
It looks like v2.2.0 is close to complete… any idea when it might be released to Maven?

**Comment by wing328** [comment]
@earth2marsh it will be released "soon" (I don't have an exact date...)

File: samples/client/petstore-security-test/javascript-closure-angular/API/Client/FakeApi.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+ * @param {!angular.$http} $http
+ * @param {!Object} $httpParamSerializer
+ * @param {!angular.$injector} $injector
+ * @struct
+ */
+API.Client.FakeApi = function($http, $httpParamSerializer, $injector) {
+  /** @private {!string} */
+  this.basePath_ = $injector.has('FakeApiBasePath') ?
+                   /** @type {!string} */ ($injector.get('FakeApiBasePath')) :
+                   'https://petstore.swagger.io  &#39; \&quot; &#x3D;end/v2  &#39; \&quot; &#x3D;end';
```
Comment (achew22) [inline samples/client/petstore-security-test/javascript-closure-angular/API/Client/FakeApi.js]: This looks kind of odd to me. Do you want the escaping in there?

Comment (achew22) [inline samples/client/petstore-security-test/javascript-closure-angular/API/Client/FakeApi.js]: Oh... I just read the file path and I understand. Sorry about that.

**Comment by achew22** [comment]
@wing328 Sorry for the slow response. Looks good to me!

**Comment by wing328** [comment]
@achew22 no problem and thanks for reviewing the fix.

**Comment by wing328** [comment]
@earth2marsh stable version v2.2.0 has been released: https://github.com/swagger-api/swagger-codegen/releases/tag/v2.2.0

**Pull request merged**